examples/component/.gitignore

```diff
@@ -1,2 +0,0 @@
-example_component
```

Comment:
I think there is a lot of value in having something you can actually run and see work however. If we do decide to remove I think it should be a seperate conversation in a different PR